src/template.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
+    />
+    <script
+      defer
+      src="https://kit.fontawesome.com/667d7efdcc.js"
+      crossorigin="anonymous"
+    ></script>
+
+    <title>Movie Slider</title>
+  </head>
+  <body>
+    <div class="container">
+      <div class="navbar">
+        <h2 class="logo">Movie concert</h2>
+        <ul class="nav-list">
+          <li class="nav-item movies">Movies</li>
```

Comment:
- [ ]  Your counter function should show the number of items(home) as well at home page. but it is now showing zero. 
![image](https://github.com/husush23/capstone--2/assets/43877689/629b9e13-a037-45ac-b314-7703b17fc4b9)

Reply:
@ishimwezachee  in part please make, because works for me look the below snapshot
![Screenshot 2023-06-02 221846](https://github.com/husush23/capstone--2/assets/114058667/ad22c2af-77a4-4ef1-8db4-5bd80c8460fe)